index.html

```diff
@@ -205,6 +205,23 @@ <h1 class="about">About Myself</h1>
         </div>
     </section>
 
-    
+    <section class="contact-form">
+        <div class="form-1">
+            <img src="images/contact form shapes-mobile.svg" alt="" class="form-image">
+            <h1 class="Contact">Contact me</h1>
+            <p class="application">If you have an application you are interested 
+                in developing, a feature that you need built or a project that needs coding. 
+                I'd love to help with it </p> 
+
+            <form action="https://formspree.io/f/{form_id}" method="post">
+                <input name="name" id="name" type="name" placeholder="Yeremias" maxlength="30" required="required">
```

Comment:
- [ ] The input type for the `name` input should be `text` and not `name`. Kindly correct this. Also, the correct way to make the form _**required**_ is to type it and not assign the string `"required"` to it. In the end, the input control for the name should look like this:

`<input name="name" id="name" type="text" placeholder="Yeremias" maxlength="30" required>` 👍 😎